SamplifyAPIClient/Response.cs

```diff
@@ -127,6 +127,13 @@ public class DetailedLineItemReportResponse : Response
         public DetailedLineItemReport Data { get; set; }
     }
 
+    [DataContract]
+    public class QuotaCellReponse : Response
+    {
+        [DataMember(Name = "data")]
+        public DetailedLineItemReport Data { get; set; }
```

Comment:
Should this contain report?

Reply:
Have updated it, thanks for pointing it out.